### ISSUE TITLE
[BUG] Test failures:  azurelinuxagent.ga.policy.regorus.PolicyError: [PolicyError] Attempted to enable policy enforcement, but feature is not supported on  architecture aarch64

### ISSUE THREAD

**Issue by waldiTM** [Azure/WALinuxAgent#3219]
**Describe the bug: A clear and concise description of what the bug is.**

```
>       raise PolicyError(msg)
E       azurelinuxagent.ga.policy.regorus.PolicyError: [PolicyError] Attempted to enable policy enforcement, but feature is not supported on  architecture aarch64

azurelinuxagent/ga/policy/policy_engine.py:123: PolicyError
```

**Distro and WALinuxAgent details (please complete the following information):**
Debian 13

**Comment by mgunnala** [comment]
@waldiTM which specific tests are failing, and what architecture are you running the tests on?

**Comment by waldiTM** [comment]
Don't you want to test the behaviour of your code, regardless of the execution environment? So as long as the code does architecture checks, the test needs to run once for each architecture value it could encounter.

As mentioned in the exception, this runs on a Linux reporting `aarch64` as architecture.

**Comment by mgunnala** [comment]
Removed the architecture check, you shouldn't see test failures anymore.

**Comment by narrieta** [comment]
fixed by #3222

**Comment by narrieta** [comment]
fixed by #3222
